src/routes/Flashcards.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const router = express.Router();
+
+const {
+    getUserFlashcards,
+    getFlashcard,
+    createFlashcard,
+    updateFlashcard,
+    deleteFlashcard
+} = require('../controllers/Flashcards');
+
+router.route('/').post(createFlashcard).get(getUserFlashcards);
+router.route('/:id').get(getFlashcard).delete(deleteFlashcard).patch(updateFlashcard);
```

Comment:
[HTTP Conventions]
Honestly, people follow the rest conventions very very loosely. But generally if you did a PATCH here, you shouldn't require all the fields. Only the ones they want to change.

So either, we should change the controller to allow partial updates or change this to a POST.

Reply:
I removed the if statement from the updateFlashcard controller, where all fields were required, therefore I'll leave the PATCH method here.